### PR TITLE
Initial benchmark

### PR DESCRIPTION
**Pull request by dominikbuenger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+to the dataset folder. See [Data](#data) for details.
+
+### Remix at a specific SNR
+
+In order to remix the dataset at a specific signal-to-noise ratio (SNR), add `--remix-snr-db ${SNR_DB}` to the
+arguments to `python3 benchmark.py` and replace `${SNR_DB}` with your SNR in dB.
+
+The mixer separates the clean data from the noise, scales the noise, and mixes them together again. The scaling is done
+in such a way that
+$$ \frac{\mathrm{power}(\mathrm{clean})}{\mathrm{power}(\mathrm{noise})} = 10^{\mathrm{SNR_DB} / 10}. $$
```
Comment (mrrostam) [inline README.md]: this doesn't render correctly by GitHub

File: README.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+## Results
+
+### STOI
+
+![](results/plots/interspeech2020_bar_plot.png)
+
+|     Engine      | Interspeech2020 |
+|:---------------:|:---------------:|
+| Mozilla RNNoise |      0.925      |
```
Comment (mrrostam) [inline README.md:119]: ~~Can you explain a bit more about these values? are there any connection between them and the plot above?~~

okay :) I think I figured it out after reading the code and looking at the results. Still believe it's better to make it clear that these scores correspond to the original data set without remixing

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.idea
```
Comment (mrrostam) [inline .gitignore:1]: Unrelated to this pr, but I noticed the license file on master has `Copyright [yyyy] [name of copyright owner]`. It's good to fix it as well

File: README.md (hunk 1 above)
Comment (dominikbuenger) [reply README.md]: Right, fixed

File: README.md (hunk 2 above)
Comment (dominikbuenger) [reply README.md:119]: You're right, I moved the plot to a different subsection, but I didn't add an additional explanation for now. Let me know if you feel that would be warranted

**Review by mrrostam — APPROVED** [review]

**Pull request merged**